package.json

```diff
@@ -3,14 +3,14 @@
   "version": "1.8.1-SNAPSHOT",
   "description": "The Open MCT core platform",
   "devDependencies": {
+    "@babel/eslint-parser": "^7.16.3",
```

Comment:
Given that this rules may revision independent of our package.json, would you mind fixing these version numbers without the ^? We use dependabot to bump the dependencies

Reply:
Is the intent of `dependency-name: "eslint*"` to update babel/eslint stuff too? Or just package literally eslint packages?

Reply:
@trusktr filed this fix here https://github.com/nasa/openmct/issues/4642

Reply:
dependencies in this PR pinned.